### PR TITLE
Feature/reorganize tasks

### PR DESCRIPTION
**Pull request by faneshion**
Reorganize the contents of the paper list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,219 +12,234 @@
...
-    - [MCAN](#mcan)
-    - [MIX](#mix)
-    - [HiNT](#hint)
+Text matching is a core component in many natural language processing tasks, where many task can be viewed as a matching between two texts input.
 
+​							$$\text{Match}(s, t) = g(f(\psi(s), \phi(t)))$$
 
-### Models
-------
+Where $s$ and $t$ are source text input and target text input, respectively. The $\psi$ and $\phi$ are representation function for input $s$ and $t$, respectively. The $f$ is the interaction function, and $g$ is the aggregation function. The representative matching tasks are as follows:
```
Comment (yangliuy) [inline README.md]: Can we add a link of the survey on the neural ranking models here to further explain this formula ? [Survey](https://arxiv.org/abs/1903.06902)

File: README.md [hunk 2]
```diff
@@ -12,219 +12,234 @@
...
 
-A Dual Embedding Space Model for Document Ranking.
-[website](https://arxiv.org/abs/1602.01137) | [paper](https://arxiv.org/pdf/1602.01137.pdf)
+### Ad-hoc Information Retrieval
 
+---
 
-#### DSSM:
-![status](artworks/ready.svg)
+**Information retrieval** (**IR**) is the activity of obtaining information system resources relevant to an information need from a collection. Searches can be based on full-text or other content-based indexing.  Here, the **Ad-hoc information retrieval** refer in particular to text-based retrieval where the number of queries is huge. Some benchmark datasets are listed in the following,
```
Comment (yangliuy) [inline README.md]: The term ad-hoc refers to the scenario where documents in the collection remain relatively static while new queries are submitted to the system continually. (cited from the [survey](https://arxiv.org/pdf/1903.06902.pdf) )

**Review by yangliuy — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -12,219 +12,249 @@
...
+* [**Gov2**](https://trec.nist.gov/data/terabyte.html) is a large Web collection where the pages are crawled from .gov. It consists of 25 million documents in total. The queries are accumulated over TREC Terabyte Tracks 2004, 2005, and 2006. There are 150 queries in total.
 
-A Latent Semantic Model with Convolutional-Pooling Structure for Information Retrieval (2014b).
-[website](https://www.microsoft.com/en-us/research/publication/a-latent-semantic-model-with-convolutional-pooling-structure-for-information-retrieval/) | [paper](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/cikm2014_cdssm_final.pdf)
+* [**MSMARCO Passage Reranking**](http://www.msmarco.org/dataset.aspx) provides a large number of information question-style queries from Bing's search logs. There passages are annotated by humans with relevant/non-relevant labels. There are 8,841822 documents in total. There are 6,980 queries and 48,598 queries for validation and test, respectively.
```
Comment (yangliuy) [inline README.md]: How about the training set?

File: README.md [hunk 4]
```diff
@@ -12,219 +12,249 @@
...
+### Community Question Answer
 
-Modeling Diverse Relevance Patterns in Ad-hoc Retrieval
-[website](https://arxiv.org/abs/1805.05737) | [paper](https://arxiv.org/pdf/1805.05737.pdf)
+---
+
+**Community Question Answer** is to automatically search for relevant answers among many responses provided for a given question (Answer Selection), and search for relevant questions to reuse their existing answers (Question Retrieval).
+
+Some benchmark datasets are listed in the following,
+
```
Comment (yangliuy) [inline README.md:211]: We can add a passage as follows for TRECQA data here:

[**TRECQA**] TREC QA data set is created by  [Wang et. al.](http://www.aclweb.org/anthology/D07-1003)   from TREC QA track 8-13 data, with candidate answers automatically selected from each question’s document pool using a combination of overlapping non-stop word counts and pattern matching. This data set is one of the most widely used benchmarks for [answer sentence selection](https://aclweb.org/aclwiki/Question_Answering_(State_of_the_art)).

**Review by yangliuy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (faneshion) [reply README.md]: Good~

File: README.md [hunk 5]
```diff
@@ -12,219 +12,249 @@
...
+1. **DSSM** [[code]](https://github.com/NTMC-Community/MatchZoo/blob/master/matchzoo/models/dssm.py) [[pdf]](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/cikm2013_DSSM_fullversion.pdf) ![status](artworks/ready.svg)
 
-#### ARC-I
-![status](artworks/ready.svg)
+   Learning Deep Structured Semantic Models for Web Search using Clickthrough Data. *CIKM 2013*.
+   [website](https://www.microsoft.com/en-us/research/publication/learning-deep-structured-semantic-models-for-web-search-using-clickthrough-data/) | [paper](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/cikm2013_DSSM_fullversion.pdf) | [tutorial](https://www.microsoft.com/en-us/research/wp-content/uploads/2017/07/dl-summer-school-2017.-Jianfeng-Gao.v2.pdf)
 
-Convolutional Neural Network Architectures for Matching Natural Language Sentences.
-[website](https://arxiv.org/abs/1503.03244) | [paper](https://arxiv.org/pdf/1503.03244.pdf)
+2. **CDSSM**  [[code]](https://github.com/NTMC-Community/MatchZoo/blob/master/matchzoo/models/cdssm.py) [[pdf]](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/www2014_cdssm_p07.pdf) ![status](artworks/ready.svg)
```
Comment (pl8787) [inline README.md:75]: Duplicate link in `[pdf]` and `paper`. 
`[code] [paper] [website]` is better.

**Review by pl8787 — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (yangliuy) [reply README.md:75]: Agree. pdf and paper are duplicated. There are some missing links in the pdf field.

Comment (faneshion) [reply README.md:75]: It's really duplicate. It will be removed.

**Comment by bwanglzu** [comment]
@faneshion I need to tweek the gramma a little bit.

**Pull request merged**